README.md

```diff
@@ -154,7 +161,7 @@ action in the same repository.
 steps:
   - name: Checkout
     id: checkout
-    uses: actions/checkout@v3
+    uses: actions/checkout@v4
```

Comment:
```suggestion
    uses: actions/checkout@vX.X.X # Set to the latest version
```

Reply:
I think a lot of folks will copy/paste and having it pinned to `v4` might still be helpful I think. It will comprise all `v4.x.x` versions and if/when we hit `v5` it can always be updated.